src/pages/BorrowerProfile/FundedBorrowerProfile.vue

```diff
@@ -1,6 +1,6 @@
 <template>
-	<www-page>
-		<div v-if="typeof loan !== 'undefined'">
+	<www-page class="relative">
```

Comment:
Should this be `tw-relative`?

Reply:
Yeah, I've already removed it because it wasn't necessary at all, thanks Casey!